src/OVAL/probes/unix/process58-capability.h

```diff
@@ -34,5 +36,10 @@ static const struct oscap_string_map CapabilityType[] = {
 {31, "CAP_SETFCAP"},
 {32, "CAP_MAC_OVERRIDE"},
 {33, "CAP_MAC_ADMIN"},
+{34, "CAP_SYS_PACCT"},
```

Comment:
comments missing in the enumeration IMO, mark it clearly which caps belong where.